lib/malloy/client/controller.hpp

```diff
@@ -19,6 +34,20 @@ namespace malloy::client
         class connection_plain;
     }
 
+    namespace detail {
+        
+        struct default_resp_filter {
```

Comment:
This could use some short comment describing it's functionality/characteristics/purpose/behavior.

Reply:
Is there an `internal` group or something I can put it into? Since its meant to be part of the API (but has to be because header files)

Reply:
I was wondering the same thing :p
Currently there is no such namespace but personally I think that `detail` might be a bit inaccurate here.

If a user is not supposed to ever use this directly I think that `internal` is a better namespace for this.

Reply:
I was thinking about doxygen :p, but in terms of namespacing I went with `detail` because thats what most libraries I've come across use. I can change it to `internal` here but imo it would be better to do that across the whole codebase to maintain consistency (since I've used `detail` in #17 as well)

Reply:
Agreed, let's keep `detail`.